components/Feature/Services/index.js

```diff
@@ -99,135 +124,179 @@ const Services = ({
   };
-  const clickCategory = e => {
-    document.getElementsByName('refer-details').forEach(x => x.removeAttribute('open'));
-
-    setResultsTitle(e);
-    setOpenReferralForm({});
-    setFeedbackSubmitted(false);
-
-    window.location.href = '#search-results-divider';
-
-    sendDataToAnalytics({
-      action: getUserGroup(referrerData['user-groups']),
-      category: CATEGORY_CATEGORIES,
-      label: e
-    });
-    setFilteredResources(categorisedResources[categorisedResources.findIndex(x => x.name === e)]);
-    setWordsToHighlight([]);
-  };
 
   return (
     <>
-      <form onSubmit={handleSearch}>
-        <label htmlFor="keyword-search">
-          <h1 id="search-for-support-header" className={`govuk-heading-l`}>
-            Search for support
-          </h1>
-        </label>
-        <input
-          id="keyword-search"
-          data-testid="keyword-search"
-          list="input-tags"
-          type="search"
-          name="search-input"
-          placeholder="Type here what you are looking for..."
-          className="govuk-input govuk-input--width-30"
-        />
-        <button
-          type="submit"
-          className="govuk-button govuk-!-margin-left-2"
-          data-testid="keyword-search-button">
-          Search
-        </button>
-      </form>
-      <h1 id="resources-header" className={`govuk-heading-l`}>
-        Explore categories
-      </h1>
-      <Categories
-        categorisedResources={categorisedResources}
-        clickCategory={clickCategory}
-        selectedCategory={filteredResources?.name}
-      />
-      <div className="govuk-grid-column-full"></div>
-      <div className="govuk-grid-column-full-width">
-        <hr
-          id="search-results-divider"
-          className={`govuk-section-break govuk-section-break--m govuk-section-break--visible ${styles['horizontal-divider']}`}
-        />
-        {filteredResources && (
-          <div key={`search-result-${filteredResources.id}`} data-testid="search-results-container">
-            <h1 id="search-results-header" className={`govuk-heading-l`}>
-              Search results
-            </h1>
-            <h2
-              data-testid="search-results-header"
-              className={`${styles['search-results-header']}`}>
-              {filteredResources.name}
-            </h2>
-
-            <div className="govuk-!-margin-bottom-6">
-              <p>
-                If the results don't contain a service or information you require, please let us
-                know.
-              </p>
-              {!feedbackSubmitted ? (
-                <Details id="feedback-summary" title="Provide feedback">
-                  <form id="search-feedback-form" onSubmit={sendFeedback}>
-                    <div className="govuk-inset-text">
-                      <div>
-                        <label htmlFor="search-feedback">Search results feedback</label>
+      <div class="govuk-grid-row">
```

Comment:
can we change all the class= to className= ?

Reply:
Yup, done